src/Components/ProjectTile.js

```diff
@@ -0,0 +1,63 @@
+import Button from "@mui/material/Button";
+import Service from '../Service';
+
+//
+//       Component: ProductTile
+//       Description: This component wraps the project in its separate box with an upvote and downvote.
+//
+//       Inputs:
+//           - NA
+//       Outputs:
+//          - NA
+const ProjectTile = ({ products, index, setProducts }) => {
+
+  const capitalizeFirstLetter = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
+  const deleteDiv = (id) => {
+    const el = document.getElementById(id);
+    Service.remove('/'+id+'/delete')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8494981)
```suggestion
    Service.remove("/"+id+'/delete')
```